src/components/collections/ProgramCollection.tsx

```diff
@@ -436,6 +436,7 @@ const composeCollectionData = (data: hasura.GET_PROGRAM_COLLECTION): ProgramData
         listPrice: pp.list_price,
         salePrice: pp.sale_price,
         soldAt: pp.sold_at && new Date(pp.sold_at),
+        publishedAt: pp.published_at,
```

Comment:
FIX: should transform to date if not null, because the type you definition is date or null
like above soldAt